### PR TITLE
better map icon logic

### LINKED ISSUE #3100: Legend Icons not appearing
**Issue Description:** View icons not appearing in the legend of web interface

* **Dynmap Version:** latest - v3.1-beta-3
* **Server Version:** spigot 1.16.2

The issue is that the web interace is look for the icon file names like: block_{world}_{view}.png.  When the file names do not have the world name in them.  

Not sure if this an intentional change, but I am getting around it by copying the files and renaming them to make the web interace happy.

### PR DESCRIPTION
**Pull request by Kimax89**
This changes the way maps should be named in worlds.txt and how the sidebar shows the icons.

Added missing map icons.

### CODE REVIEW

**Comment by Kimax89** [comment]
This has been tested and is currently in use at:

http://dynmap.wattup.eu

**Comment by SiphMC** [comment]
@Kimax89 
Do you load your map in hires? Also I would like to use it because I got problem with dynmap anyways, if you want to help, I posted in the issue.

Does it support towny, I guess, but I just want to be sure.

File: DynmapCore/src/main/resources/extracted/web/js/map.js [hunk 1]
```diff
@@ -278,12 +278,17 @@ DynMap.prototype = {
 					worldsadded[wname] = true;
 				}
 
+				var worldName = wname;
+				if(wname.startsWith('world_')) {
```
Comment (mikeprimm) [inline DynmapCore/src/main/resources/extracted/web/js/map.js:282]: This does not work with non-default world names (world is just the default)

**Review by mikeprimm — CHANGES_REQUESTED** [review]
Needs to handle non-default base world name ('world' is just default.

File: DynmapCore/src/main/resources/extracted/web/js/map.js (hunk 1 above)
Comment (Kimax89) [reply DynmapCore/src/main/resources/extracted/web/js/map.js:282]: No problem.

Will take another look at it, soon.

**Review by mikeprimm — APPROVED** [review]
Looks good!

**Pull request merged**